.github/workflows/check-git-log.yml

```diff
@@ -0,0 +1,69 @@
+name: Check Git Log for EMIS Cassettes (PII)
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  check-git-log:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Run git log command
+      run: |
+        LOG=$(git log --all --oneline -- spec/support/vcr_cassettes/emis)
```

Comment:
did we determine that `-all` shouldn't be included here?

Reply:
It should be. We want to check all branches/tags not just the one that's being pushed.

You should only omit `--all` locally when you're checking a single branch.